### ISSUE TITLE
Accessibility: Object Status: There is no speech output when object status icons are accessed in virtual pc mode.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3445]
Description: There is no speech output when object status icons are accessed in virtual pc mode.

Expected: Each & every UI should be read out, when accessed. 

Screenshot:
![image](https://user-images.githubusercontent.com/32538291/94520546-164e5580-024a-11eb-86e8-3214432f82ab.png)

![image](https://user-images.githubusercontent.com/32538291/94520559-19e1dc80-024a-11eb-9f43-063c0800a170.png)

![image](https://user-images.githubusercontent.com/32538291/94520571-1e0dfa00-024a-11eb-8954-f5d244824f53.png)

**Comment by Lokanathan-k** [comment]
fixed as part of the PR https://github.com/SAP/fundamental-ngx/pull/4176
